vcd/sample_vcd_test_config.json

```diff
@@ -90,6 +90,44 @@
     "uploadPieceSize": 5,
     "uploadProgress": true
   },
+  "testEnvBuild": {
+    "//": "This section is optional for running tests. It's only needed to build the test environment in a new vCD",
+
+    "// ": "Entities that will be created in an empty vCD use names defined above",
+
+    "// ": "vcd.org                    -> organization name",
```

Comment:
I can't find a better line where to put this comment, but it would be really nice to make all sample resource names consistent. Now they are all over the place:

```
    "org": "myorg",
    "vdc": "myvdc",
...
      "name": "my-catalog-name",
      "catalogItem": "my-catalog-item"
...
    "edgeGateway": "tfEdgeGW",
...
      "name": "my-provider-vdc",
      "storageProfile": "my-storage-profile-name",
      "networkPool": "my-network-pool"
...
    "externalNetwork": "my-ext-network",
```

I suggest using the format we agreed on in the test templates, i.e.:
`tf_<some_name>`